### PR TITLE
enable turbolinks5

### PR DESCRIPTION
**Pull request by samjacobclift**
**headlines**

1) Enable turbolinks again, see (https://github.com/librariesio/libraries.io/issues/1055)

**TODOs**

- [ ] It seems that that event is fired multiple times per link click, so need to check that out

### CODE REVIEW

**Comment by samjacobclift** [comment]
I might redo the branch rather that resolve the conflicts :)

**Comment by andrew** [comment]
@samjacobclift need to remove the existing GA tracking in the header of application.html.erb I think

**Comment by andrew** [comment]
Also got some conflicts to resolve but thanks for jumping on this so quickly! ✨

**Comment by samjacobclift** [comment]
@andrew no worries I like learning about turbolinks! its an interesting thing!

**Comment by andrew** [comment]
We may need to review all the other javascript that's loaded on doc ready to be sure it doesn't need to be fired again on turbolinks:load

**Comment by samjacobclift** [comment]
I think so too, there are a few duplicate page views I've been noticing

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -4,7 +4,7 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="msvalidate.01" content="3667D2FEB238C1BF174F3CB1AE0D3C16" />
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track' => true %>
```
Comment (samjacobclift) [inline app/views/layouts/application.html.erb]: I think these data-turbolinks-track attrs are causing extra track events, feels like there are a few more elsewhere too

Comment (samjacobclift) [inline app/views/layouts/application.html.erb]: hmm now looking at it think they can be left in they shouldn't be doing any harm, that being said though something is causing extra 'pageview' events from being fired. The platform page produced some consistently for me

**Comment by andrew** [comment]
@samjacobclift I found what the problem was, turbolinks assumes your javascript to be in the head of the page, adding `, 'data-turbolinks-eval': false` to the script tag at the bottom of the layout fixed the repeated calls to GA, going to deploy this now 🚂

**Pull request merged**